indico/htdocs/js/indico/modules/category_management.js

```diff
@@ -0,0 +1,92 @@
+/* This file is part of Indico.
+ * Copyright (C) 2002 - 2016 European Organization for Nuclear Research (CERN).
+ *
+ * Indico is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License as
+ * published by the Free Software Foundation; either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * Indico is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Indico; if not, see <http://www.gnu.org/licenses/>.
+ */
+
+(function() {
+    'use strict';
+
+    $(document).ready(function() {
+        setupCategoriesTable();
+    });
+
+    function setupCategoriesTable() {
+        var $table = $('table.categories-management');
+        var $tbody = $table.find('tbody');
+        var $categoryRowSelector = 'tr[data-category-id]';
```

Comment:
I'm not a fan of `$` prefixes but for this one it's wrong for sure since it's not even a jQuery object but just a string